src/TsTypes.ts

```diff
@@ -3,30 +3,33 @@ import { camelCase, upperFirst } from 'lodash'
 export namespace TsType {
 
   export interface TsTypeSettings {
-    declareSimpleType?: boolean
-    declareReferenced?: boolean
-    useFullReferencePathAsName?: boolean
+    declarationDescription?: boolean
     // TODO declareProperties?: boolean
-    useInterfaceDeclaration?: boolean
-    endTypeWithSemicolon?: boolean
+    declareReferenced?: boolean
+    declareSimpleType?: boolean
+    exportInterfaces?: boolean
     endPropertyWithSemicolon?: boolean
-    declarationDescription?: boolean
+    endTypeWithSemicolon?: boolean
     propertyDescription?: boolean
+    useConstEnums?: boolean
+    useFullReferencePathAsName?: boolean
+    useInterfaceDeclaration?: boolean
   }
 
   export var DEFAULT_SETTINGS: TsTypeSettings = {
     declarationDescription: true,
+    // declareProperties: false,
     declareReferenced: true,
     declareSimpleType: false,
     endPropertyWithSemicolon: true,
     endTypeWithSemicolon: true,
     propertyDescription: true,
+    useConstEnums: false,
```

Comment:
I want to avoid generating code by default, unless there's a strong reason otherwise. If this causes confusion for people down the line, we can always switch it back.